lib/src/over_react_test/jacket.dart

```diff
@@ -72,6 +74,8 @@ class TestJacket<T extends react.Component> {
   bool _isMounted = false;
 
   void _render(over_react.ReactElement reactElement) {
+    // ignore: invalid_use_of_visible_for_testing_member
+    currentComponentZone = Zone.current;
```

Comment:
Can we add comments to these explaining why we're doing it? Or split it out into a method that has a doc comment.